### ISSUE TITLE
Add so you can update an Add-On

### ISSUE THREAD

**Issue by McTwist** [McTwist/Blockland-Repository#28]
Currently you can only upload once and never update an Add-On.

First off, you should be able to update an Add-On through edit, and it will throw you to the edit page.

Secondly, if you just upload a file, it will check that file toward the database and guess which Add-On it belongs to and try to update it, if you own it. Then it will throw you to the edit page instead of the upload page.

The edit page should contain so you can pick channel and version and force you to pick a version higher the current one. If done correctly, how you upload will automatically throw you here nonetheless.

**Comment by McTwist** [comment]
~~Blocked by #33.~~ Due to a separate view, this is not blocked anymore.

**Comment by McTwist** [comment]
I would like to make so when you update an Add-On, it should take you to a different view that can compare them more easily.

**Comment by McTwist** [comment]
This is solved from d4683d24ab5d91067941a42b1ad6c46d9ae4f5ae. Any bugs that is raised need a new issue.
